SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+The latest three major release versions of Nextcloud are currently being supported with security updates.
+Please visit https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule for further details.
+
+## Reporting a Vulnerability
+
+Security is very important to us. If you have discovered a security issue with Nextcloud,
+please read our responsible disclosure guidelines and contact us at [hackerone.com/nextcloud](https://hackerone.com/nextcloud).
+Your report should include:
+
+- Product version
+- A vulnerability description
+- Reproduction steps
+
+A member of the security team will confirm the vulnerability, determine its impact, and develop a fix.
```

Comment:
Does this applies to all repositories under the `nextcloud` namespace? I'm thinking VM, Docker, or community apps like Polls or News? Maybe state what's included or ask and make sure these repositories to have their own `SECURITY.md` file?

Reply:
That is a fair point. https://hackerone.com/nextcloud?type=team lists what is in scope and what is not. I'd refrain from copying this info as it will get out of sync sooner or later.

@LukasReschke @nickvergessen how would you prefer to handle this?

Reply:
Maybe simply:
```suggestion
If in scope of the project a member of the security team will confirm the vulnerability, determine its impact, and develop a fix.
Otherwise the team will contact the maintainer in a secure way and make sure the issue gets fixed.
```